index.html

```diff
@@ -0,0 +1,152 @@
+    <section class="container headline">
+        <h1>Meri Gogichashvili</h1>
+        <h2>Software Developer</h2>
+            <p>I can help you build a product , feature or website Look through some of my work a
+                nd experience! If you like what you see and have a project you need coded, don’t 
+                hesitate to contact me.
+            </p>
+    <hr>
+    <div class="info-wrapper">
+        <ul>
+            <li class="list-header">Language</li>
+            <li class="list">Javascript</li>
+            <li class="list">Ruby</li>
+            <li class="list">Html</li>
+            <li class="list">Css</li>
+        </ul>
+        <ul>
+            <li class="list-header">Frameworks</li>
+            <li class="list">Bootstrap</li>
+            <li class="list">Ruby on Rails</li>
+            <li class="list">RSPeC</li>
+            <li class="list">CapyBara</li>
+            <li class="list">Selenium</li>
+        </ul>
+        <ul>
+            <li class="list-header">Skills</li>
+            <li class="list">Codekit</li>
+            <li class="list">Github</li>
+            <li class="list">Codepen</li>
+            <li class="list">Gitlab</li>
+            <li class="list">Terminal</li>
+        </ul>
+</div>
+    <hr>
+        <div class="flex-icons">
+            <ul>
+                <li>
+                    <a href="#" title="icons"><i class="fab fa-github"></i></a>
+                </li>
+                <li>
+                    <a href="#" title="icons"><i class="fab fa-linkedin-in"></i></a>
+                </li>
+                <li>
+                    <a href="#" title="icons"><i class="fab fa-angellist"></i></a>
+                </li>
+                <li>
+                    <a href="#" title="icons"><i class="fab fa-twitter"></i></a>
+                </li>
+                <li>
+                    <a href="#" title="icons"><i class="fab fa-medium-m"></i></a>
+                </li>
+            </ul>
+        </div>
+</section> 
+<div class="works-wrapper">
+    <section class="container works">
+        <div class="work-images">
+            <img src="images/Group55.png" alt="down-arrow">
+            <img src="images/Work.png" alt="work">
+        </div>
+        <h2>My Recent Work</h2>
+        <hr>
+        <img class="placeholder" src="images/placeholder.png" alt="image-placeholder">
+        <h3>Multi-Post Stories</h3>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups 
+            required. has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.</p>
+        <div class="grid-container">
+            <div class="item1">css</div>
+                <hr class="vl">
+            <div class="item1">html</div>
+                <hr class="vl">
+            <div class="item1">bootstrap</div>
+                <hr class="vl">
+            <div class="item1">Ruby</div>
+        </div>
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them a `li`, for instance:

```
<ul>
  <li>CSS</li>
  <li>HTML</li>
  <li>Bootstrap</li>
  ..and so on...
</ul>
```

- [ ] Apply the list `ul` with `li` elements  (the same as the change required above) for each project you are showing the list of project technologies.